File: Two Pointer/Reverse_String.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        reverse(s.begin(),s.end());
    }
};


/* Using Two Pointer Technique */

class Solution {
public:
    void reverseString(vector<char>& s) {
        int len = s.size();
        int left = 0,right = len-1;
        
        while(left <= right)
        {
            swap(s[left],s[right]);
            left++;
            right--;
        }
    }
};

File: Two Pointer/Java/Easy/ReverseString.java
class Solution {
    public void reverseString(char[] s) {
        int left =0, right = s.length-1;
        while(left < right)
        {
            char temp = s[left];
            s[left] = s[right];
            s[right] = temp;
            left++;
            right--;
        }
    }
}

/* ----- 
    If the return type is string : String str = new StringBuilder(new String(s)).reverse().toString();
*/